src/main/java/org/prgrms/vouchermanagement/voucher/repository/FileVoucherRepository.java

```diff
@@ -54,8 +60,11 @@ public Optional<Voucher> findById(UUID voucherId) {
     @Override
     public Voucher save(Voucher voucher) {
 
-        try (BufferedWriter writer = new BufferedWriter(new FileWriter(new File(path), true));) {
-            writer.write(parseCsvFormat(voucher.getVoucherId().toString(), voucher.getVoucherType().name(), String.valueOf(voucher.getDiscountAmount())));
+        try (BufferedWriter writer = new BufferedWriter(new FileWriter(new File(path), true))) {
```

Comment:
```suggestion
        try (BufferedWriter writer = new BufferedWriter(new FileWriter(path, true))) {
```
fileName을 파라미터로 바로 받을 수 있네요